task/pins/index.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs')
+const util = require('util')
+const path = require('path')
+
+const { parseIcon, combinePictoPin, getColoredPin } = require('../../lib/svg_icon');
+
+const excludeList = [
+  // transports
+  'aerialway',
+  'airport',
+  'bus',
+  'entrance',
+  'ferry',
+  'parking',
+  'parking-bike',
+  'rail',
+  'rail-light',
+  'rail-metro',
```

Comment:
Don't we need to use these icons as pins too ? For example, one could search for a list of "parking" items.

Reply:
I guess so. I went for the easiest route as these ones have a blue picto on white background, which is an exception and probably wasn't thought about by the design. I can re-include all transports as white on blue for now. Would it be ok?